tests/NuGet.Services.AzureSearch.Tests/BaseDocumentBuilderFacts.cs

```diff
@@ -160,6 +161,83 @@ public void AllowNullDependencyGroups()
 
                 Assert.Null(full.FlattenedDependencies);
             }
+
+            [Fact]
+            public void IfLeafHasIconFile_LinksToFlatContainer()
+            {
+                var leaf = new PackageDetailsCatalogLeaf
+                {
+                    PackageId = Data.PackageId,
+                    PackageVersion = Data.NormalizedVersion,
+                    IconFile = "iconFile",
+                    IconUrl = "iconUrl"
+                };
+
+                var full = new HijackDocument.Full();
+
+                Target.PopulateMetadata(full, Data.NormalizedVersion, leaf);
+
+                Assert.Equal(
+                    $"{Data.FlatContainerBaseUrl}{Data.FlatContainerContainerName}/{Data.PackageId.ToLowerInvariant()}/{Data.NormalizedVersion.ToLowerInvariant()}/icon",
+                    full.IconUrl);
+            }
+
+            [Fact]
+            public void IfLeafDoesNotHaveIconFile_UsesIconUrl()
+            {
+                var iconUrl = "iconUrl";
```

Comment:
Nit change: why do you define a local variable for "IconUrl" here but for the test above, you directly set "IconUrl" at the constructor.